pynetdicom3/applicationentity.py

```diff
@@ -141,9 +143,11 @@ def on_c_store(dataset):
     transfer_syntaxes : List of pydicom.uid.UID
         The supported transfer syntaxes
     """
+
+
     # pylint: disable=too-many-instance-attributes,too-many-public-methods
     def __init__(self, ae_title='PYNETDICOM', port=0, scu_sop_class=None,
-                 scp_sop_class=None, transfer_syntax=None):
+                 scp_sop_class=None, transfer_syntax=None, tls_args=None):
```

Comment:
should `tls_args` be on `__init__` or `associate`?

Reply:
I'd prefer it be a keyword arg for `AE.associate()` and `AE.start()`.

I've been thinking about changing the AE init to remove port and having it as an `associate` keyword with a default of 0 and a `start` arg.